docs/examples/en/controls/OrbitControls.html

```diff
@@ -267,11 +267,21 @@ <h3>[property:Float zoomSpeed]</h3>
 
 		<h2>Methods</h2>
 
-		<h3>[method:undefined dispose] ()</h3>
+		<h3>[method:undefined connect] ()</h3>
```

Comment:
The function is missing the `domElement` parameter.

Reply:
Good catch, should be fixed now!